prompt_pure_setup

```diff
@@ -0,0 +1 @@
+pure.zsh
```

Comment:
1. How about moving these into a folder called `fpath` to keep the project directory cleaner?
2. Could you turn these into sourced files instead of symlinks? See [here](https://github.com/mafredri/zsh-async/blob/5248bd85bf2c522ebfa2b8d18ee7bc736d10c791/async.plugin.zsh) for an example (https://github.com/mafredri/zsh-async/pull/22#issuecomment-384260603) for discussion.

Regarding 2, symlinks can be disabled in git and as such, won't work for everyone.